config/manifests/bases/oadp-operator.clusterserviceversion.yaml

```diff
@@ -9,14 +9,26 @@ metadata:
     createdAt: "2020-09-08T12:21:00Z"
     description: OADP (OpenShift API for Data Protection) operator sets up and installs
       Data Protection Applications on the OpenShift platform.
-    features.operators.openshift.io/token-auth-aws: "true"
     olm.skipRange: '>=0.0.0 <99.0.0'
     operatorframework.io/suggested-namespace: openshift-adp
-    operators.openshift.io/infrastructure-features: '["Disconnected"]'
     operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
       Platform Plus"]'
     repository: https://github.com/openshift/oadp-operator
     support: Red Hat
+    # DEPRECATED but still functional starting with OCP v4.14
+    operators.openshift.io/infrastructure-features: '["Disconnected"]'
+    # New infrastructure annotation format
+    # MUST provide either the string value (not boolean) "true" or "false" for each
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "false"
```

Comment:
We are proxy aware.
https://github.com/kaovilai/oadp-operator/blob/37fd7fba1d3593786826c81e446fdb0fe364cde7/controllers/velero.go#L413